react-app/src/components/projects/ProjectDetail.tsx

```diff
@@ -219,6 +219,19 @@ const ProjectDetail = (props: IProjectDetail) => {
     }
   };
 
+  const showNotes = (note) => {
+    // noteId 2 are SRES only notes
+    if (note.NoteTypeId == 2 && !isAdmin) {
```

Comment:
Should auditors be allowed to see this as well?
Also, would like to use the enum type if possible.
```suggestion
    if (note.NoteTypeId == NoteTypes.PRIVATE && !(isAdmin || isAuditor)) {
```